x-pack/filebeat/module/rabbitmq/log/_meta/fields.yml

```diff
@@ -0,0 +1,17 @@
+- name: log
+  type: group
+  description: >
+    RabbitMQ log files
+  fields:
+    - name: pid
```

Comment:
This should be mapped to `process.pid` i think: https://www.elastic.co/guide/en/ecs/current/ecs-process.html

Reply:
Note the example value `<0.222.0>`. These aren't process ids in the OS sense (i.e. a value you can pass to `kill`, etc). They're an [erlang-specific way of rendering process ids](https://stackoverflow.com/questions/243363/can-someone-explain-the-structure-of-a-pid-in-erlang).

Since these are strings (not longs) and can't be used like other pid values, it doesn't seem like they should be ECS values

Reply:
Got it, thanks for the clarification.